### ISSUE TITLE
Nodemon

### ISSUE THREAD

**Issue by SunJieMing** [DevMountain/Siri#7]
`npm install --save-dev nodemon` won't make it so they can run `nodemon`.  They have to install it globally: `npm install -g nodemon`.

**Comment by IV-R** [comment]
Correct! If someone installs nodemon using npm install --save-dev nodemon, it will only be available as a dev dependency within the project, and it will need to be executed through npm scripts or using npx nodemon.

To run nodemon globally from the terminal without specifying it in a project, they need to install it globally using:

npm install -g nodemon

Once installed globally, they can simply run nodemon from any directory without needing to install it again for each project.
